src/main/java/eu/deltasource/internship/abankingsystem/model/BankInstitution.java

```diff
@@ -0,0 +1,84 @@
+package eu.deltasource.internship.abankingsystem.model;
+
+import eu.deltasource.internship.abankingsystem.enums.ExchangeRate;
+import eu.deltasource.internship.abankingsystem.enums.Taxes;
+
+import java.util.*;
+
+public class BankInstitution {
+
+    private final String name;
+
+    private final String address;
+
+    private final ArrayList<Owner> customers = new ArrayList<>();
+
+    private final Map<Taxes, Double> priceList; // (that describes all taxes and fees and all currency exchange rates)
+
+    private final Map<ExchangeRate, Double> exchangeRates;
+
+    private final List<BankAccount> bankAccounts = new ArrayList<>();
+
+    private int dayCountTime = 1;
```

Comment:
I don't get the point of this

Reply:
Was used to automatically set of LocalDate when creating a Transaction. Will be removed due to required usage of LocalDateTime for the task and manual setup for transaction time for dates. Will be done ASAP.